dlio_benchmark/checkpointing/pytorch_checkpointing.py

```diff
@@ -42,14 +59,16 @@ def __init__(self):
         super().__init__("pt")
 
     @dlp.log
-    def get_tensor(self, size):
-        return torch.randint(high=1, size=(size,), dtype=torch.int8)
+    def get_tensor(self, size, datatype="int8"):
+        return torch.ones(size, dtype=get_torch_datatype(datatype))
```

Comment:
I had tried torch.ones, the tensor representation in memory was smaller as PyTorch was doing some optimization. that's why I used randint. Can you make sure the I/O and memory representation remain same?

Reply:
sounds good, can you fix the size variable to be length?